test/test_simulation_store.jl

```diff
@@ -139,21 +143,23 @@ function _verify_data(expected, store, model, name, time, columns)
     expected_df = DataFrames.DataFrame(expected, columns)
     df = read_result(DataFrames.DataFrame, store, model, name, time)
     @test expected_df == df
-
-    # TODO-DT read_result with DenseAxisArray
 end
 
 @testset "Test SimulationStore 2-d arrays" begin
     sim = Dict(
         "initial_time" => Dates.DateTime("2020-01-01T00:00:00"),
         "step_resolution" => Dates.Hour(24),
-        "num_steps" => 2,
+        "num_steps" => 50,
     )
     variables = Dict(
         PSI.VariableKey(ActivePowerVariable, ThermalStandard) =>
             Dict("cache_priority" => CachePriority.HIGH, "keep_in_cache" => true),
+        PSI.VariableKey(ActivePowerVariable, RenewableDispatch) =>
+            Dict("cache_priority" => CachePriority.HIGH, "keep_in_cache" => true),
         PSI.VariableKey(ActivePowerVariable, InterruptibleLoad) =>
             Dict("cache_priority" => CachePriority.LOW, "keep_in_cache" => false),
+        PSI.VariableKey(ActivePowerVariable, RenewableFix) =>
+            Dict("cache_priority" => CachePriority.LOW, "keep_in_cache" => false),
```

Comment:
Are we even using the CachePriority at the moment? Is it worth to remove that in this branch?

Reply:
Sounds good. If we ever need it, we can easily add it back.